go/cmd/zk/zkcmd.go

```diff
@@ -456,13 +467,15 @@ func cmdTouch(ctx context.Context, subFlags *flag.FlagSet, args []string) error
 	return nil
 }
 
-func cmdRm(ctx context.Context, subFlags *flag.FlagSet, args []string) error {
+func cmdRm(ctx context.Context, subFlags *pflag.FlagSet, args []string) error {
 	var (
 		force             = subFlags.Bool("f", false, "no warning on nonexistent node")
 		recursiveDelete   = subFlags.Bool("r", false, "recursive delete")
 		forceAndRecursive = subFlags.Bool("rf", false, "shorthand for -r -f")
```

Comment:
here, if you do this:

```go
force = subFlags.BoolVarP("force", "f", ...)
recursive = subFlags.BoolVarP("recursive", "r", ...)
```

then pflag will let you combine short opts, so the following all work:

```
zk rm -f --recursive
zk rm --force -r
zk rm -f -r
zk rm -rf
```